frontend/www/js/omegaup/components/problem/CollectionDetails.test.ts

```diff
@@ -10,11 +10,27 @@ describe('CollectionDetails.vue', () => {
     const wrapper = shallowMount(collection_Details, {
       propsData: {
         data: {
-          type: 'author',
+          type: 'problemLevelBasicIntroductionToProgramming',
+          collection: <string[]>[
+            'problemTagMatrices',
+            'problemTagDiophantineEquations',
+            'problemTagInputAndOutput',
+            'problemTagArrays',
+          ],
```

Comment:
El tipo de collection, no es `string[]`, así que lo puedes cambiar por esto:
```suggestion
          collection: [
            { alias: 'problemTagMatrices' },
            { alias: 'problemTagDiophantineEquations' },
            { alias: 'problemTagInputAndOutput' },
            { alias: 'problemTagArrays' },
          ],
```
?